bridgeql/django/urls.py

```diff
@@ -7,14 +7,18 @@
 except ImportError:
     from django.conf.urls import url as path
 
-from bridgeql.django.bridge import read_django_model
+from bridgeql.django import bridge
 from bridgeql.django.settings import bridgeql_settings
 from bridgeql.django.views import index, generate_bridgeql_schema
 
 bridgeql_settings.validate()
 
 urlpatterns = [
-    path('reader/', read_django_model, name='bridgeql_django_read'),
+    path('reader/', bridge.read_django_model, name='bridgeql_django_read'),
```

Comment:
Let's have standard practice

CRUD - create/read/update/delete
create - POST
read - GET

- add a test case for big enough params to get raise "HTTP 413 Content Too Large" exception and return the same
- thinking, if there will be any such case to get these type of real requests.
- what will be the solution in the case, should we allow to use POST ?

update - PATCH/PUT (do not create new record)
delete - DELETE